src/main/java/com/lgtm/easymoney/models/Account.java

```diff
@@ -3,17 +3,17 @@
 import com.lgtm.easymoney.configs.DBConsts;
 import java.io.Serializable;
 import javax.persistence.*;
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+
+import lombok.*;
```

Comment:
lets only import modules we used, checkstyle complains about this shit i believe